libcudacxx/include/cuda/std/detail/__access_property

```diff
@@ -129,19 +129,21 @@
  * (v. August 20, 2021)
  */
 
+#include <cuda_runtime_api.h>
+
 _LIBCUDACXX_BEGIN_NAMESPACE_CUDA
 
 namespace __detail_ap
 {
 
 _CCCL_HOST_DEVICE constexpr uint32_t __ap_floor_log2(uint32_t __x)
 {
-  return (__x == 1 | __x == 0) ? 0 : 1 + __ap_floor_log2(__x >> 1);
+  return ((__x == 1) || (__x == 0)) ? 0 : 1 + __ap_floor_log2(__x >> 1);
```

Comment:
NOTE: MSVC was warning about those lines and I believe the && is much clearer given that we are dealing with booleans